1 table sketch.sql

```diff
@@ -0,0 +1,21 @@
+/*
+musicrentalDB
+
+MusicRental
+    MusicRentalID int not null PK
+    CustomerFirstName varchar (100) not null, not blank
+    CustomerLastName varchar(100) not null, not blank
+    Instrument varchar(100) not null, not blank
+    Category varchar(100) not null, not blank
+    InstrumentID computed column concat first 3 letters of the instrument and of the category
+    MonthlyRentalFee decimal (6,2), not null, must be greater than 0
+    TotalProfit computed column MonthlyRentalFee * months rented
+    DateRented Date not null, not future date
+    DateReturned not null not future date
+    DateInserted datetime not null, default getdate()
+    
+    multy constraint columns
+        DateReturned mus be after data rented 
```

Comment:
Please implement the above constraint in your table file

Reply:
sorry forgot to implement my multi constraints column

Reply:
no problem, good job